src/pages/AddProperty/AddProp.jsx

```diff
@@ -125,8 +126,8 @@ class AddProp extends React.Component {
                 <TopNav />
                 <div className="saveDraft-publish-buttons">
                     <button
-                        onClick={() => this.setState({draft: true})}>
-                        Save As Draft</button>
+                        onClick={this.submitProperty(true)}
+                    >Save As Draft</button>
                     <button id="pubButton">Publish Listing</button>
```

Comment:
Attach that same function to the publish button and set it to false. When you click the button does it work as expected? No errors in the console?